rllib/env/single_agent_episode.py

```diff
@@ -323,7 +339,9 @@ def concat_episode(self, episode_chunk: "SingleAgentEpisode") -> None:
             from both episodes.
         """
         assert episode_chunk.id_ == self.id_
-        assert not self.is_done and not self.is_finalized
+        # NOTE (sven): This is what we agreed on. As the replay buffers must be
```

Comment:
Makes sense! Let's just remove the note. It's clear now that also finalized episodes must be "concatenable".